### ISSUE TITLE
[Accessibility] In display mode, users cannot navigate between form fields and read their values

### ISSUE THREAD

**Issue by SjsJane** [surveyjs/survey-library#8209]
T17934 - [Accessibility] Display Mode - This functions as vertical radiogroup but does not read radio buttons, when in the view mode for users
https://surveyjs.answerdesk.io/internal/ticket/details/T17934
***
Consider the following demo: [View Plunker](https://plnkr.co/edit/IJalUwNCvC683Z60). The issue is that in display mode, users cannot tab and focus form fields. Therefore, a screen reader cannot announce form field values.

**Comment by dmitry-kurmanov** [comment]
dev:

`isInputReadonly`:
- [ ] library
- [ ] creator

**Comment by JaneSjs** [comment]
+1 It's necessary to allow screen readers to read values of read-only fields.
[T18420 - Make an input field readonly instead of disabled](https://surveyjs.answerdesk.io/internal/ticket/details/T18420)
